src/Sentry.NLog/LogLevelExtensions.cs

```diff
@@ -29,7 +29,7 @@ internal static class LogLevelExtensions
                     return SentryLevel.Warning;
 
                 default:
-                    return null;
+                    return SentryLevel.Info;
```

Comment:
Don't see any tests dealing with the change of behavior here. Not sure what is the impact of this change now.

Reply:
Well `null` is not a good default value. Prefer something more sensible. Though it cannot be tested :)

Reply:
Can revert the change if you want the `null`-value.

Reply:
Unless we know what's the side effect here I'd rather leave it as it was.
I agree generally but in this case, a value that cannot be mapped might just not fit `Info`. It would be a value created in the future which we don't support (i.e: `None`).

Reply:
Reverted the changes. And forced pushed them away.